third_party/silabs/BUILD.gn

```diff
@@ -35,17 +36,25 @@ declare_args() {
   sl_matter_version_str = ""
 }
 
-assert(efr32_sdk_target != "", "efr32_sdk_target must be specified")
 
 config("silabs_config") {
   if (sl_matter_version_str != "") {
     defines = [ "SL_MATTER_VERSION_STRING=\"${sl_matter_version_str}\"" ]
   }
 }
 
-group("efr32_sdk") {
-  public_deps = [ efr32_sdk_target ]
-  public_configs = [ ":silabs_config" ]
+if (silabs_board == "BRD4325B") {  # CCP board
+  assert(siwx917_sdk_target != "", "siwx917_sdk_target must be specified")
+  group("siwx917_sdk") {
+    public_deps = [siwx917_sdk_target ]
+    public_configs = [ ":silabs_config" ]
+  }
+} else {
+  assert(efr32_sdk_target != "", "efr32_sdk_target must be specified")
+  group("efr32_sdk") {
+    public_deps = [ efr32_sdk_target ]
+    public_configs = [ ":silabs_config" ]
+  }
 }
 
 if (silabs_board != "BRD4325B") {  # CCP board
```

Comment:
Seems this part could potentially go in the else instead of having two conditionnals?

Reply:
done in latest commit